### PR TITLE
[WIP] Enable PSR-4 and namespaces

### PR DESCRIPTION
**Pull request by petk**
Hello, this patch is a work in progress to enable PSR-4 autoloading and namespaces in this library... More changes soon...

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -39,8 +39,8 @@
         "pear-horde/Horde_Text_Diff": "2.*"
     },
     "autoload": {
-        "psr-0": {
-            "Horde_Text_Diff": "lib/"
+        "psr-4": {
+            "Horde": "lib/Horde/"
```
Comment (peter279k) [inline composer.json:43]: PSR-4 namespace definition is incorrect. I think it should be `Horde\\`.

**Review by peter279k — CHANGES_REQUESTED** [review]
I think this PR is the big change and this namespce changes into the `PR-4`.

I think it's required because the `PSR-0` namespace is deprecated from PHP-FIG.

And the Travis CI build is failed due to the error namespace definition.

**Comment by yunosh** [comment]
This is going happen to for all libaries at once, once Horde 6 has been released.
Unless you intend to prepare PRs for all packages, which would be very much appreciated, I'm going to close this individual PR.

**Comment by petk** [comment]
Thank you for the response. Is there any ETA or roadmap for the mentioned Horde 6 in this case?

**Comment by yunosh** [comment]
Not yet, but this year :)

**Comment by petk** [comment]
Ok, looking forward when this will be available. PSR-4 is sort of standard these days... Thank you.

**Pull request closed without merging**